docs/guest.md

```diff
@@ -50,6 +50,10 @@ Please refer to [velero-plugin-for-vsphere tags](https://github.com/vmware-tanzu
 
 If it is an air-gapped environment, please refer to [Install Notes with Customized Images](install-notes.md) for extra required steps.
 
+#### Install with self-signed certificate
```

Comment:
Would you please also add the same section in supervisor.md and clearly call out it is not supported on supervisor cluster?